taskcoordinator/app/src/main/java/com/microservices/taskcoordinator/dto/LaundryStateDto.java

```diff
@@ -21,9 +21,13 @@
     private Integer version;
 
     public LaundryStateDto(LaundryStateEntity laundryStateEntity) {
-        this.laundryId = laundryStateEntity.getId();
+        this.id = laundryStateEntity.getId();
         this.queueWaitingTime = laundryStateEntity.getQueueWaitingTime();
         this.reservedTime = laundryStateEntity.getReservedTime();
         this.version = laundryStateEntity.getVersion();
     }
+
+    public long getCompletionTimePrediction() {
```

Comment:
It is more then weird to place prediction logic in dto.

Reply:
yeah, i agree, but where should i move it? it's not convenient to have it entity (for now), cause i work with dtos mostly
hm